Assets/Mirror/Migration/Editor/Scripts.cs

```diff
@@ -22,6 +22,7 @@ public class Scripts : MonoBehaviour {
                 "NetworkInstanceId",
                 "GetNetworkSendInterval()",
                 "NetworkServer.connections",
+                "NetworkServer.connections.Values.Values",
```

Comment:
Don't know if that's possible (from my latest research, I failed to find the solution) to use regex instead of some kind of duplicate. Anyway, your solution is far better than nothing